yarn-project/bot/src/factory.ts

```diff
@@ -101,7 +101,7 @@ export class BotFactory {
       const address = account.getAddress();
       this.log.info(`Deploying account at ${address}`);
 
-      const claim = await this.bridgeL1FeeJuice(address, 10n ** 22n);
+      const claim = await this.bridgeL1FeeJuice(address, MINT_AMOUNT);
```

Comment:
Would we need it to mint itself a couple of times, or do we get sufficient to cover our fees for some time? Just thinking that it would be annoying if we run into the issue where at block 4000 it runs out of juice.